interface/main/calendar/add_edit_event.php

```diff
@@ -1460,6 +1487,23 @@ function find_available(extra) {
         ?>
         </span>
             </div>
+            <?php
+            // Lifemesh button only show if module is enabled
```

Comment:
Same issue as above. Here are basically rendering below patient field, so maybe something like:
```
$eventDispatcher->dispatch(ApptRenderEvent::RENDER_BELOW_PATIENT, new ApptRenderEvent($row), 10);

```
(and then the module can send all the needed output if pertinent)

Reply:
I respect the idea of making the system as generic as possible. I was following the pattern that Jerry did here
https://github.com/openemr/openemr/blob/6a6397d56af40818f792e3ff4af598c7ab04edda/interface/patient_file/report/patient_report.php#L272

It looked to me that this line of code is only activated if the fax module is installed.  This is why I set up a qualifier for this dispatch.

Reply:
That's odd, I don't remember seeing that with the $oefax, but I'll open up an issue as we should remove the if statement.  We'll need to update the fax module to deal with the global flag.

Reply:
Question:
For the Lifemesh project, can we push the refactoring of the code to Phase 2 of this build so that they can get this into the patch without more delays? I will make sure to refactor the code to include the proper use of the dispatcher and remove the qualifying if statements. 

They have a timeline that they would like to meet.

Reply:
Hi @juggernautsei , just wondering if we should make a new repository for this module and use @adunsulag 's [module  installer](https://github.com/openemr/oe-module-installer-plugin) for bringing the code in